release-notes/release-notes59.md

```diff
@@ -45,6 +45,81 @@ For information on:
 - The `content_langs` option, see: [Content appearance options - `content_langs`]({{site.baseurl}}/configure/localization/#content_langs).
 - The `Lang` editor command, see: [Commands available for {{site.productname}} - Core Editor commands]({{site.baseurl}}/advanced/editor-command-identifiers/#coreeditorcommands).
 
+### New table commands
+
+A series of new commands have been added to the table plugin to improve the usability of the plugin.
+- The new command `mceTableCellToggleClass` allows the user to add or remove a class to selected cells .
+- The new command `mceTableToggleClass` allows the user to add or remove a class to selected tables.
```

Comment:
```suggestion
- The new command `mceTableToggleClass` allows the user to add or remove a class to the selected table.
```

Reply:
For some reason Github isn't letting me comment on the line below but it needs fixing too